apps/api/test/functional/deployment-fallback.spec.ts

```diff
@@ -0,0 +1,316 @@
+import Long from "long";
+
+import { closeConnections } from "@src/core";
+import { app, initDb } from "@src/rest-app";
+
+import { createAkashBlock, createDay, createDeployment, createDeploymentGroup, createDeploymentGroupResource, createTransaction } from "@test/seeders";
+import { createAkashAddress } from "@test/seeders/akash-address.seeder";
+
+describe("Deployment Fallback API", () => {
+  const now = new Date();
+  const date = now.toISOString().split("T")[0];
+
+  afterAll(async () => {
+    await closeConnections();
```

Comment:
this is not needed https://github.com/akash-network/console/blob/main/apps/api/test/setup-functional-tests.ts#L39